packages/larva-tokens/src/brands/variety-compat.json

```diff
@@ -0,0 +1,108 @@
+{
+  "imports": [
+    "../base/all.json",
```

Comment:
In the PHP side, are we enqueuing this and the original VY tokens file? or only this one? If we are enqueuing both, we might be able to remove these imports – they will bloat the CSS variables file and that might not be necessary since we only need a few values to be overridden. If we are only using the compact, then it makes sense to include them so it is comprehensive. Let me know if that doesn't make sense!

Reply:
Right now it is only enqueuing the compat tokens if compat_css is true.

Reply:
Left a note in the other PR too - I think we can remove these imports. Can you try that out, please? It's just _a lot_ of custom properties in the file that we don't need, and it will make debugging harder in the future.